src/api/frontend-config/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface FrontendConfig {
+  backendUrl: string,
```

Comment:
There's a tailing comma although no element is following.

Reply:
removed